app/serializers/web_hook_user_serializer.rb

```diff
@@ -38,6 +38,9 @@ def staff_attributes(*attrs)
     use_logo_small_as_avatar
     pending_posts_count
     status
+    display_sidebar_tags
+    sidebar_category_ids
+    sidebar_tags
```

Comment:
Excluding these from the web hook as I don't think they will be useful in anyway.